modules/core/src/matrix_operations.cpp

```diff
@@ -1057,6 +1083,29 @@ void cv::reduce(InputArray _src, OutputArray _dst, int dim, int op, int dtype)
             else if(sdepth == CV_64F && ddepth == CV_64F)
                 func = reduceMinR64f;
         }
+        else if( op == CV_REDUCE_SUM2 )
+        {
+            if(sdepth == CV_8U && ddepth == CV_32S)
+                func = GET_OPTIMIZED(reduceSum2R8u32s);
```

Comment:
Remove `GET_OPTIMIZED()`.
There is no corresponding function anyway.

Also changes in "ReduceFunc" broke all corresponding calls (changes can be reverted, see corresponding comment).